README.md

```diff
@@ -58,10 +58,6 @@ module "eks-jx" {
   source  = "jenkins-x/eks-jx/aws"
 }
 
-output "jx_requirements" {
```

Comment:
should not be deleted. It's a case of missing parens, as you had identified before.

Reply:
I took it out of here because it was redundant -- as long as it's still mentioned in the broader instructions, right?

Reply:
I think that makes sense.